src/VisualBasic/CodeCracker/Refactoring/ChangeAnyToAllCodeFixProvider.vb

```diff
@@ -0,0 +1,139 @@
+﻿Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.CodeFixes
+Imports System.Composition
+Imports System.Collections.Immutable
+Imports Microsoft.CodeAnalysis.CodeActions
+Imports System.Threading
+Imports Microsoft.CodeAnalysis.VisualBasic
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Microsoft.CodeAnalysis.Formatting
+
+Namespace Refactoring
+    <ExportCodeFixProvider(LanguageNames.VisualBasic, Name:=NameOf(ChangeAnyToAllCodeFixProvider)), [Shared]>
+    Public Class ChangeAnyToAllCodeFixProvider
+        Inherits CodeFixProvider
+
+        Public Overrides ReadOnly Property FixableDiagnosticIds As ImmutableArray(Of String)
+            Get
+                Return ImmutableArray.Create(DiagnosticId.ChangeAnyToAll.ToDiagnosticId(),
+                                             DiagnosticId.ChangeAllToAny.ToDiagnosticId())
+            End Get
+        End Property
+
+        Public NotOverridable Overrides Function GetFixAllProvider() As FixAllProvider
+            Return WellKnownFixAllProviders.BatchFixer
+        End Function
+
+        Public Overrides Async Function RegisterCodeFixesAsync(context As CodeFixContext) As Task
+            Dim diag = context.Diagnostics.First
+            Dim message = If(diag.Id = DiagnosticId.ChangeAnyToAll.ToDiagnosticId, "Change Any to All", "Change All to Any")
+            context.RegisterCodeFix(CodeAction.Create(message, Function(c) ConvertAsync(context.Document, diag.Location, c)), diag)
+            Await Task.Yield
+        End Function
```

Comment:
Why await `Task.Yield`? This has an impact on the execution. I suggest you remove the `Async` modifier from the method and return `Task.FromResult(0)` as on the original method.